superset-frontend/src/cccs-viz/plugins/plugin-chart-email-renderer/src/plugin/buildQuery.ts

```diff
@@ -46,7 +46,7 @@ export default function buildQuery(formData: QueryFormData): QueryContext {
   return buildQueryContext(modifiedFormData, (baseQueryObject) => [
     {
       ...baseQueryObject,
-      row_limit: 10,
+      row_limit: 2,  // Only 1 row is permitted for email rendering
```

Comment:
It is 2 because we want to make sure that there is only 1 row returned by the query. if 2 show up then we know that we are not filtering for a single email so we won't render anything. If we changed this to 1 then it would always render the first email it gets back from trino which could be an option?

Reply:
Ah-ha! This makes sense. I think the error is the cleaner way to go. Thanks for the explanation! :)